libs/checkpoint/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@langchain/langgraph-checkpoint",
-  "version": "0.0.6",
+  "version": "0.0.7",
```

Comment:
npm released version is `0.0.7`, I didn't manually make this change so not sure how it got here?